src/app/containers/ArticleTimestamp/helpers.js

```diff
@@ -14,18 +14,22 @@ export const isSameDay = (dayToCompare, timestamp) => {
 
 export const isToday = (timestamp) => isSameDay(Date.now(), timestamp);
 
-export const formatType = ({ lastPublished = null, firstPublished }) => {
+export const formatType = ({
+  lastPublished = null,
+  firstPublished,
+  datetimeLocale,
```

Comment:
Do we want to set a default locale here?

Reply:
`en-gb`?